content/geoip/updating-databases.mdx

```diff
@@ -169,10 +186,6 @@ Or a `wget` command like the following:
 wget -S --method HEAD --user=YOUR_ACCOUNT_ID --password=YOUR_LICENSE_KEY 'https://download.maxmind.com/geoip/databases/GeoIP2-City-CSV/download?suffix=zip'
 ```
 
-Where `YOUR_ACCOUNT_ID` is a placeholder for your account’s actual account ID
-and `YOUR_LICENSE_KEY` is a placeholder for your account’s actual license key.
-[Learn more about license keys on our knowledge base](https://support.maxmind.com/hc/en-us/articles/4407116112539-Using-License-Keys).
```

Comment:
It might make sense to have this here as well. I suspect most people just jump the relevant section and may miss that this information is above.